js/npm/etherscan/package.json

```diff
@@ -21,13 +22,16 @@
     "Parity",
     "Promise"
   ],
-  "scripts": {
+  "peerDependencies": {
+    "es6-promise": "^4.0.5",
+    "isomorphic-fetch": "^2.2.1",
+    "babel-polyfill": "^6.22.0"
```

Comment:
Scratch that.

Babel actually has the solution - the transform runtime plugin. Basically means only `babel-runtime` is added as a package dep, and the runtime includes the polyfill without polluting the global namespace. No Promise, polyfill and other include-at-head crainess. (Well, according to them - this is their library mode)